src/content/docs/dropins/cart/functions.mdx

```diff
@@ -546,6 +548,7 @@ The `UpdateProductsFromCart` object has the following parameters:
     ['Parameter', 'Type', 'Req?', 'Description'],
     ['uid', 'string', 'Yes', 'The unique ID of an item in the cart.'],
     ['quantity', 'number', 'Yes', 'The new quantity of the item.'],
+    ['optionsUIDs', 'string[]', 'No', 'The selected options for the product, such as color or size, using the unique ID for an object.']
```

Comment:
```suggestion
    ['optionsUIDs', 'string[]', 'No', 'The selected options for the product using the unique ID for an object.']
```

Reply:
maybe in here, instead of providing examples, we could provide the possible options?

Reply:
@danidelcar the possible options meaning example `optionsUIDs` numbers?

Reply:
@ericeoeur you mentioned examples like color or size, is there a list of possible options like those?, might be worth adding that. If we're talking about a long list, then no. Hope it helps!

Reply:
I might look into adding a link to that info, if we have it, but for the moment, we can go ahead with this PR